pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2425443865>#694c2e</a><span> - 10/21/2024, 08:18 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The hardcoded configuration path '/root' may cause issues in non-Docker environments and reduce flexibility. Consider implementing a more flexible configuration loading strategy.
</div>
<div id="fix">
<b>Fix:</b> Implement a flexible configuration loading strategy that checks multiple paths and supports environment-specific configuration.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -28,7 +28,13 @@
  		viper.SetConfigName("config")
  		viper.SetConfigType("yaml")
 -		viper.AddConfigPath("/root")
 +		// Check multiple paths for the configuration file
 +		viper.AddConfigPath("/root")           // Docker environment
 +		viper.AddConfigPath(".")              // Current directory
 +		viper.AddConfigPath("$HOME/.config")  // Home directory
 +		viper.AddConfigPath("../../")         // Project root (for development)
 +
 +		// You can also use an environment variable to specify the config path
 +		viper.AutomaticEnv()
 
  		if err := viper.ReadInConfig(); err != nil {
  			panic(err)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged